tests/core/full_node/test_full_node.py

```diff
@@ -2190,3 +2190,73 @@ async def test_long_reorg(
     peak = node.full_node.blockchain.get_peak()
     assert peak.header_hash != chain_2_peak
     assert peak.weight > chain_2_weight
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("light_blocks", [True, False])
+@pytest.mark.parametrize(
+    "chain_length",
+    [
+        0,
+        # TODO: understand why this breaks
+        #    900
+    ],
+)
+async def test_long_reorg_nodes(
```

Comment:
add a test for this but with no reorg and ensure nodes can sync without new blocks being added